config/routes.rb

```diff
@@ -7,8 +7,10 @@
   resources :stockrooms
   devise_for :users, controllers: { registrations: "registrations" }
   root 'pages#index'
+
   get 'reports', to: 'logs#index'
-  get 'reports/stockrooms', to: 'logs#stockrooms'
+  get 'reports/stockrooms/:id/', to: 'logs#stockrooms'
```

Comment:
just adding a dynamic variable `:id` (any name is possible and it will be access by params[:id] or any name that you defined)